data/json/mutations/mutations.json

```
@@ -485,8 +485,7 @@
     "id": "GOODMEMORY",
     "name": { "str": "Good Memory" },
     "points": 3,
```

```suggestion
    "points": 2,
```
Ditto, particularly if not using skill rust.

---

I wonder if there's a way to adjust point value depending on Skill Rust's setting.

---

Because that's the *only* thing Good Memory affects.

---

Good point. That actually makes it simple - if skill rust is not in use, the trait won't be taken (possibly a note that it's only applicable with skill rust on?), so the cost is only for when skill rust is in use.